cmd/xl-storage-disk-id-check.go

```diff
@@ -821,6 +829,83 @@ func (p *xlStorageDiskIDCheck) monitorDiskStatus() {
 	}
 }
 
+// monitorDiskStatus should be called once when a drive has been marked offline.
+// Once the disk has been deemed ok, it will return to online status.
+func (p *xlStorageDiskIDCheck) monitorDiskWritable(ctx context.Context) {
+	const (
+		// We check every 15 seconds if the disk is writable and we can read back.
+		checkEvery = 15 * time.Second
+
+		// Disk has 2 minutes to complete write+read.
+		timeoutOperation = 2 * time.Minute
+
+		// If the disk has completed an operation successfully within last 5 seconds, don't check it.
+		skipIfSuccessBefore = 5 * time.Second
+	)
+
+	t := time.NewTicker(checkEvery)
+	defer t.Stop()
+	fn := mustGetUUID()
```

Comment:
We can always write new file, allowing filesystem to allocate a new block in some sense?.

Reply:
I meant a new filename in essence, creates a new namespace entry on the fs.

Reply:
@harshavardhana Don't want to deal with potential buildup, so IMO overwriting should be fine. Should do plenty of writes to ensure we are testing disk reasonably.

Reply:
Okay